### ISSUE TITLE
Add A Quick Settings Panel

### ISSUE THREAD

**Issue by MatteCrystal** [mgmeyers/obsidian-style-settings#27]
**Proposed enhancement** 

The style settings plugin is awesome, however, it has one fatal flaw. You need to navigate to settings, style settings plugin, open the corresponding dropdowns, and choose your settings. A very long and laborious task to say the least.

Ok maybe it's not as laborious as I make it sound, especially since most settings will be set once and never touched again. However, this process restricts some pretty interesting use cases.

For instance, I have a theme that uses Andy's sliding panes. The option to allow panels to stack is awesome and can be very helpful in viewing in fullscreen mode and looking over notes (as seen below). 

![image](https://user-images.githubusercontent.com/32424764/136130136-0bae5b5e-bc1e-4df0-a0ec-82c1ec25ebeb.png)

But if I snap obsidian to use half my computer screen try to edit 2 pages side by side you can see there isn't enough space to fully see both screens. Very frustrating. 

![image](https://user-images.githubusercontent.com/32424764/136130174-990c0558-2bff-404b-bc87-ea74bb7e03f4.png)

In these scenarios, it makes sense to go into the style settings and temporarily disable the stacking panes option. This will cause the pages to perfectly fit side by side when obsidian is snapped to half my screens resolution. 

![image](https://user-images.githubusercontent.com/32424764/136130260-ee888b85-a424-46a0-ac75-f473be7b5aad.png)

As you can see this sort of setting isn't a one-and-done sort of thing. I'll constantly be flicking it on and off. So the process listed above is way too long to be a workable solution. 

TLDR: We need a quicker way to change certain options. If changing a specific setting was 1 click away it could open up the doors to a lot of cool things. It could also help speed up the sometimes tedious process of going back and forth opening settings panel, changing settings, backing out to see if the settings are ok, not liking your changes, and repeating the process again and again. 

I have 2 proposed solutions. 

**Solution 1**
The Ideal solution is to have the settings also be available in it's own pane/page/window that can be dragged around like any other page (A quickSettings panel). Then the pane would be filled only with the settings specifically chosen by the developer and user. So for example, I'm thinking you could add a quickSettings option which when set to true will indicate that that option should be displayed in the quickSetting page. 

```
id: Enable-Stacked-Panes
        title: Enable Stacked Panes
        description: Enable to allow panes/pages to stack side by side
        type: class-select
        allowEmpty: false
        quickSetting: true
        default: stackedPanes
        options:
            - 
                label: Enabled
                value: stackedPanes
            - 
                label: Disabled
                value: 
```

This is how a theme developer could choose what settings should be in the quickSettings panel by default. But It would be great if the user could go into the settings and click an add to quickSettings button. This way if there is some random setting they change often then they'll be able to also add it to the list. 

**Solution 2**
Very similar except instead of having dev and user pick desired settings you instead literally just take all options and place it into the quickSettings menu. This way would be more cluttered with settings that probably aren't as relevant in day to day use but would still make changing settings a lot quicker.

**Comment by mgmeyers** [comment]
@MatteCrystal You can now open style settings in a new pane rather than the settings modal:

<img width="710" alt="Screen Shot 2021-12-14 at 1 01 08 PM" src="https://user-images.githubusercontent.com/2694747/146078681-a599e4b1-e5cc-4ee7-82cd-bc868e37a2a0.png">


<img width="1064" alt="Screen Shot 2021-12-14 at 1 00 48 PM" src="https://user-images.githubusercontent.com/2694747/146078642-261abae3-7114-45fd-a04f-35fb4c0a54d5.png">

Let me know if this works for your use case!
